modules/ROOT/pages/osv2-faq.adoc

```diff
@@ -79,8 +79,38 @@ For this reason, data access time is the same, whether you store keys in one par
 
 == How long can data persist in Object Store v2?
 
-Object Store v2 data persists 30 days. The expiry time is reset if an update to that key occurs. If there is no update, the key is deleted at the 30 day mark.
+The maximum TTL (time to live) is 2,592,000 seconds (30 days).
 
+The `entryTtl` value, specified in the global configuration parameters for the connector,
+determines when to evict key-value pairs from the store.
+
+The TTL can be either _rolling_ or _static_:
+
+Rolling::
+Accessing the data extends the TTL for 30 days.
++
+The minimum rolling TTL is 1 second.
+The maximum rolling TTL is unlimited as long as the data continues to be accessed.
+Static::
+The value of `entryTtl` determines when the data is evicted.
++
+The minimum static TTL is 1 second.
+The maximum (and default) static TTL is 2592000 seconds (30 days).
+If the `entryTtl` value is greater than 30 days, TTL is set to the maximum value (2592000 seconds/30 days) without returning an error.
+
+If the app doesn't specify a value for `entryTtl`, the default value depends on the Mule version.
+
+* Mule versions 4.2.1 and later:
++
+Object stores have a rolling TTL by default.
++
+If the data is accessed at least once a week, the TTL is extended for 30 days.
+Otherwise, it is evicted in the next 7 to 30 days from the last expiration date.
+[NOTE]
+Rolling TTL is available for all apps using Mule version 4.2.x, 4.3.0, and later.
```

Comment:
"Mule versions 4.2.1 and later" is what is used in PR 51

Reply:
good catch